covalent_ui/webapp/src/components/common/LatticeDrawer.js

```diff
@@ -33,8 +33,9 @@ const drawerPaperStyles = (theme) => ({
   width: latticeDrawerWidth,
   boxSizing: 'border-box',
   border: 'none',
-  bgcolor: alpha(theme.palette.background.default, 0.3),
-  backdropFilter: 'blur(16px)',
+  bgcolor: alpha('#080819', 0.8),
```

Comment:
Some values are being hardcoded however not sure about the requirements around theming, should be okay for now.